library/control/src/modules/InstError.rb

```diff
@@ -83,7 +76,7 @@ def SaveLogs
       cmd = Convert.to_map(
         WFM.Execute(
           path(".local.bash_output"),
-          Builtins.sformat("save_y2logs '%1'", String.Quote(savelogsto))
+          Builtins.sformat("/usr/sbin/save_y2logs '%1'", String.Quote(savelogsto))
```

Comment:
Bad idea: %1 is already quoted. This will now be double-quoted which is most likely not what you want.